### PR TITLE
Rename role to label

### PR DESCRIPTION
**Pull request by raminqaf**
depends on: https://github.com/bakdata/kpops-examples/pull/25

### CODE REVIEW

File: kpops/components/base_components/pipeline_component.py [hunk 1]
```diff
@@ -141,11 +141,11 @@ def set_error_topic(self, topic: KafkaTopic) -> None:
         :param topic: Error topic
         """
 
-    def add_extra_output_topic(self, topic: KafkaTopic, role: str) -> None:
+    def add_extra_output_topic(self, topic: KafkaTopic, label: str) -> None:
         """Add an output topic of type extra.
 
         :param topic: Output topic
-        :param role: Role that is unique to the extra output topic
+        :param label: Role that is unique to the extra output topic
```
Comment (disrupted) [inline kpops/components/base_components/pipeline_component.py:148]: it still says role in the description 🤔

**Review by disrupted — CHANGES_REQUESTED** [review]

File: kpops/components/base_components/pipeline_component.py (hunk 1 above)
Comment (raminqaf) [reply kpops/components/base_components/pipeline_component.py]: fixed! There should be no more `role` around. Appart from the ones that are the values of the test. I don't know if we want to change those

File: kpops/components/common/topic.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ class TopicConfig(DescConfigModel):
         default={}, description=describe_attr("configs", __doc__)
     )
     # TODO: We can rename this but this would be a breaking change
```
Comment (disrupted) [inline kpops/components/common/topic.py:63]: todos resolved

**Review by disrupted — CHANGES_REQUESTED** [review]

File: tests/components/streams_bootstrap_v3/test_streams_app.py [hunk 3]
```diff
@@ -143,13 +143,13 @@ def test_set_topics(self):
                         "example-input": {"type": "input"},
                         "b": {"type": "input"},
                         "a": {"type": "input"},
-                        "topic-extra2": {"role": "role2"},
-                        "topic-extra3": {"role": "role2"},
-                        "topic-extra": {"role": "role1"},
+                        "topic-extra2": {"label": "role2"},
+                        "topic-extra3": {"label": "role2"},
+                        "topic-extra": {"label": "role1"},
```
Comment (disrupted) [inline tests/components/streams_bootstrap_v3/test_streams_app.py:148]: should we rename the values as well?

**Review by disrupted — APPROVED** [review]
one question, otherwise LGTM

File: tests/components/streams_bootstrap_v3/test_streams_app.py (hunk 3 above)
Comment (raminqaf) [reply tests/components/streams_bootstrap_v3/test_streams_app.py:148]: We can, but these are just some values that don't have any meaning. Could be `foo` or `bar` or `baz`

**Pull request merged**